server/gc/src/test/java/org/apache/accumulo/gc/GarbageCollectionTest.java

```diff
@@ -51,13 +58,29 @@
     TreeMap<String,Status> filesToReplicate = new TreeMap<>();
 
     @Override
-    public boolean getCandidates(String continuePoint, List<String> ret) {
-      Iterator<String> iter = candidates.tailSet(continuePoint, false).iterator();
-      while (iter.hasNext() && ret.size() < 3) {
-        ret.add(iter.next());
+    public void processCandidates() throws TableNotFoundException, IOException {
+
+      Iterator<String> candidatesIter = candidates.iterator();
+
+      while (candidatesIter.hasNext()) {
+        List<String> candidatesBatch = readCandidatesThatFitInMemory(candidatesIter);
+        new GarbageCollectionAlgorithm().collectBatch(this, candidatesBatch);
       }
+      // Remove all candidates that were tagged for deletion now that the processing of
+      // candidates is complete for this round. This was removed from the 'delete' method
+      // due to that causing a ConcurrentModificationException. T
```

Comment:
Extra character in comment.
```suggestion
      // due to that causing a ConcurrentModificationException.
```

Reply:
Any idea why making the two type changes you suggested causes the fastbuild to fail with a formatting error?

Reply:
White space maybe? I know the fastbuild checks if you built before pushing. Did you build locally after committing the suggestions? I almost always have to due to imports or formatting.